ansible-ipi-install/roles/node-prep/tasks/40_bridge.yml

```diff
@@ -11,9 +11,9 @@
         nmcli device show {{ pub_nic }} | grep GENERAL.CONNECTION | awk '{sub(/[^ ]+[ ]+/,"")}1'
       register: pub_con_name
 
-    - name: Disconnect provisioning connection
+    - name: Disconnect {{ prov_bridge }} connection
```

Comment:
```
- name: Disconnect {{ provisioning_bridge | default('provisioning') }} connection
```
^ I think this "should" work.

Reply:
Why don't we set the default value in a fact to avoid using the default filter every time? Like we are doing here https://github.com/openshift-kni/baremetal-deploy/blob/master/ansible-ipi-install/roles/node-prep/tasks/10_validation.yml#L198-L204